unreleased.md

```diff
@@ -1 +1,6 @@
 # Unreleased
+
+## Bugfixes
+
+- The special character problem in databases of `ObjectUserType`, which is the
+  custom type we give with convention for object types, fixed.
```

Comment:
`ObjectUserType` was causing its data to be corrupted when it contains special characters, fixed.